### ISSUE TITLE
[ts] parameter --name not respected

### ISSUE THREAD

**Issue by leinardi** [jiahaog/nativefier#905]
### Description

the current ts branch doesn't seem to respect the parameter `--name`.

The name should be "Webex-Teams" but actually is "SignintoWebexTeams":
![image](https://user-images.githubusercontent.com/273338/74604874-81378b80-50c2-11ea-8419-c97f643ef3a0.png)
And also the output directory and binary names are wrong.

### Steps to reproduce issue

1. specify a name different from the website title
2. check the build output

### Details

- Are you nativefying a *public* website? yes
- Full nativefier command used to build your app: `nativefier --name "Webex-Teams" --platform linux --arch x64 --electron-version 8.0.1 --tray --single-instance --counter --bounce --honest https://teams.webex.com --verbose`
- Version of Nativefier (run `nativefier --version`): `53a92de14921103a59ae640bd1d635a91181db60`
- Version of node.js (run `node --version`): `v10.15.2`
- OS: `Ubuntu 19.10`
- Error message / stack trace (if any):

```
$ nativefier --name "Webex-Teams" --platform linux --arch x64 --electron-version 8.0.1 --tray --single-instance --counter --bounce --honest https://teams.webex.com --verbose
Running in verbose mode! This will produce a mountain of logs and is recommended only for troubleshooting or if you like Shakespeare.

Performing async options post-processing.
Inferring user agent for electron 8.0.1 / linux
Grabbing electron<->chrome versions file from https://atom.io/download/atom-shell/index.json
Inferring icon for https://teams.webex.com/ on linux
Inferring icon from store for https://teams.webex.com/ on linux
Allowed icon formats when building for linux: [ '.png', '.ico' ]
Got name Webex-Teams from options. No inferring needed
Sanitized filename for Webex-Teams : Webex-Teams
Inferring name for https://teams.webex.com/
Got 249 icons from gitcloud
Max icon match score: 0
No relevant icon in store.
Trying to extract a .png icon from the page.
Fetched 1.7 kb page at https://teams.webex.com/
Inferred title: Sign in to Webex Teams
Sanitized filename for Sign in to Webex Teams : SignintoWebexTeams
Associated electron v8.0.1 to chrome v80.0.3987.86
Given chrome 80.0.3987.86 on linux, using user agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.86 Safari/537.36
Got an icon from the page.
Writing 148.6 kb icon to /tmp/nativefier-13-27-9-iconinfer-31435eaOonpVhCKSv/icon.ico

Preparing Electron app...
Copying electron app from /home/leinardi/Workspace/github/nativefier/app to /tmp/nativefier-13-27-9-app-31435hN5boGqxb2o5
Writing app config to /tmp/nativefier-13-27-9-app-31435hN5boGqxb2o5/nativefier.json
No files to inject, skipping copy.
Updating /tmp/nativefier-13-27-9-app-31435hN5boGqxb2o5/package.json 'name' field to signintowebexteams-nativefier-ff9bb8

Converting icons...
Converting icon /tmp/nativefier-13-27-9-iconinfer-31435eaOonpVhCKSv/icon.ico to /tmp/nativefier-13-27-9-iconconv-31435yunJGok81MX2/icon.png. Calling: "/home/leinardi/Workspace/github/nativefier/lib/icon-scripts/convertToPng" "/tmp/nativefier-13-27-9-iconinfer-31435eaOonpVhCKSv/icon.ico" "/tmp/nativefier-13-27-9-iconconv-31435yunJGok81MX2/icon.png"
Failed to convert icon to .png, skipping. { stdOut: '',
  stdError:
   '/bin/sh: 1: /home/leinardi/Workspace/github/nativefier/lib/icon-scripts/convertToPng: not found\n' }

Packaging... This will take a few seconds, maybe minutes if the requested Electron isn't cached yet...
  electron-packager Electron Packager 14.2.1
  electron-packager Node v10.15.2
  electron-packager Host Operating system: linux 5.3.0-29-generic (x64) +0ms
  electron-packager Packager Options: {"arch":"x64","asar":false,"darwinDarkModeSupport":false,"dir":"/tmp/nativefier-13-27-9-app-31435hN5boGqxb2o5","electronVersion":"8.0.1","icon":"/tmp/nativefier-13-27-9-iconinfer-31435eaOonpVhCKSv/icon.ico","name":"SignintoWebexTeams","out":"/home/leinardi/Workspace/github/nativefier","overwrite":true,"platform":"linux","targetUrl":"https://teams.webex.com/","tmpdir":false,"win32metadata":{"ProductName":"Webex-Teams","InternalName":"Webex-Teams","FileDescription":"Webex-Teams"}} +1ms
  electron-packager Target Platforms: linux +0ms
  electron-packager Target Architectures: x64 +0ms
  electron-packager Inferring appVersion from version in /tmp/nativefier-13-27-9-app-31435hN5boGqxb2o5/package.json +0ms
  electron-packager Application name: SignintoWebexTeams +5ms
  electron-packager Target Electron version: 8.0.1 +0ms
  electron-packager Ignored path regular expressions: [ '/\\.git($|/)',
  '/node_modules/\\.bin($|/)',
  '\\.o(bj)?$',
  '/tmp/electron-packager' ] +0ms
  electron-packager Downloading Electron with options {"platform":"linux","arch":"x64","version":"8.0.1","artifactName":"electron"} +0ms
Packaging app for platform linux x64 using electron v8.0.1
  electron-packager Creating /home/leinardi/Workspace/github/nativefier/linux-x64-template +2ms
  electron-packager Extracting /home/leinardi/.cache/electron/httpsgithub.comelectronelectronreleasesdownloadv8.0.1electron-v8.0.1-linux-x64.zip/electron-v8.0.1-linux-x64.zip to /home/leinardi/Workspace/github/nativefier/linux-x64-template +1ms
  electron-packager Initializing app in /home/leinardi/Workspace/github/nativefier/SignintoWebexTeams-linux-x64 from /home/leinardi/Workspace/github/nativefier/linux-x64-template template +0ms
  electron-packager Ignored paths based on the out param: [ '/home/leinardi/Workspace/github/nativefier/SignintoWebexTeams-darwin-x64',
  '/home/leinardi/Workspace/github/nativefier/SignintoWebexTeams-linux-ia32',
  '/home/leinardi/Workspace/github/nativefier/SignintoWebexTeams-linux-x64',
  '/home/leinardi/Workspace/github/nativefier/SignintoWebexTeams-linux-armv7l',
  '/home/leinardi/Workspace/github/nativefier/SignintoWebexTeams-linux-arm64',
  '/home/leinardi/Workspace/github/nativefier/SignintoWebexTeams-linux-mips64el',
  '/home/leinardi/Workspace/github/nativefier/SignintoWebexTeams-mas-x64',
  '/home/leinardi/Workspace/github/nativefier/SignintoWebexTeams-win32-ia32',
  '/home/leinardi/Workspace/github/nativefier/SignintoWebexTeams-win32-x64',
  '/home/leinardi/Workspace/github/nativefier/SignintoWebexTeams-win32-arm64' ] +1s
  electron-packager Renaming electron to SignintoWebexTeams in /home/leinardi/Workspace/github/nativefier/SignintoWebexTeams-linux-x64 +116ms

Finalizing build...
Copying icons if necessary
Copying icon /tmp/nativefier-13-27-9-iconinfer-31435eaOonpVhCKSv/icon.ico to /home/leinardi/Workspace/github/nativefier/SignintoWebexTeams-linux-x64/resources/app/icon.ico
App built to /home/leinardi/Workspace/github/nativefier/SignintoWebexTeams-linux-x64

```

In the current master the build output and app name are correctly set:
```
$ nativefier --name "Webex-Teams" --platform linux --arch x64 --electron-version 8.0.1 --tray --single-instance --counter --bounce --honest https://teams.webex.com --verbose
  icons [==============================                    ] 60%Packaging app for platform linux x64 using electron v8.0.1

App built to /home/leinardi/Workspace/github/nativefier/Webex-Teams-linux-x64
```

**Comment by ronjouch** [comment]
@leinardi thanks! Fixed in dd192c749063ec54e486c40c5f8b072569f05dbe ! Thanks a lot for the testing 🙂! Keep it coming if you see more broken tests.

**Comment by leinardi** [comment]
I can confirm is fixed :+1:
